pkgs/by-name/di/dirbuster/package.nix

```
@@ -0,0 +1,74 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, makeWrapper
```

```suggestion
, makeBinaryWrapper
```

Use `makeBinaryWrapper` for a better compatibility with Darwin architectures.

---

`makeBinaryWrapper` results in the process not starting 🤔

---

Oh. OK let's leave it like that then. Weird, but ok.

---

it was due to the `binutils` dependency (for `strings`). fixed by dropping `stdenvNoCC`.

---

TIL ! Thank you !